code/game/objects/items/toys.dm

```diff
@@ -1086,3 +1087,179 @@
 	desc = "Small mechanical canary in a cage, does absolutely nothing of any importance!"
 	icon = 'icons/mob/animal.dmi'
 	icon_state = "canary"
+
+/obj/item/toy/balloon
+	name = "balloon"
+	desc = "A simple balloon."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "balloon_deflated"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/toys.dmi', "right_hand" = 'icons/mob/in-hand/right/toys.dmi')
+	w_class = W_CLASS_TINY
+	force = 0
+	throwforce = 0
+	var/col = "#FFFFFF"
+	var/inflated_type = /obj/item/toy/balloon/inflated
+
+/obj/item/toy/balloon/New(atom/A, var/chosen_col)
+	..(A)
+	if(col)
+		if(chosen_col)
+			col = chosen_col
+		else
+			col = rgb(rand(0,255),rand(0,255),rand(0,255))
+		color = col
+		update_icon()
+
+/obj/item/toy/balloon/update_icon()
+	overlays.len = 0
+	var/image/shine_overlay = image('icons/obj/toy.dmi', src, "[icon_state]_shine")
+	shine_overlay.appearance_flags = RESET_COLOR
+	overlays += shine_overlay
+
+/obj/item/toy/balloon/attack_self(mob/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/haslungs = FALSE
+		for(var/I in H.internal_organs)
+			if(istype(I, /datum/organ/internal/lungs))
+				haslungs = TRUE
+		if((H.species && H.species.flags & NO_BREATHE) || !haslungs)
+			to_chat(user, "You can't blow up \the [src] without lungs!")
+			return
+	inflate(user)
+
+/obj/item/toy/balloon/proc/inflate(mob/user, datum/gas_mixture/G)
+	var/obj/item/toy/balloon/inflated/B = new inflated_type(get_turf(src), col)
+	if(user)
+		user.drop_item(src, force_drop = 1)
+		user.put_in_hands(B)
+		to_chat(user, "You blow up \the [src].")
+	playsound(src, 'sound/misc/balloon_inflate.ogg', 50, 1)
+	if(!G)
+		B.air_contents = new /datum/gas_mixture()
+		B.air_contents.volume = 12 //liters
+		B.air_contents.temperature = T20C
+		B.air_contents.carbon_dioxide = 0.5
```

Comment:
Every species *does* exhale CO2.
But I'll change it just in case that changes in the future.